### PR TITLE
style the column content

### PR DESCRIPTION
**Pull request by leggsimon**
This adds the styling for the column content, you can use this by wrapping the content in a containing `div.myft-section__column-content`. Have fun !! 🎉 

### Here, have some images
----------------------

#### < M
screenshot 2017-03-15 11 54 30

#### M < XL
screenshot 2017-03-15 11 54 45

#### > XL
screenshot 2017-03-15 11 54 58

### CODE REVIEW

**Comment by GlynnPhillips** [comment]
😍 screenshots

File: client/components/v2/alerts/section.scss [hunk 1]
```diff
@@ -51,3 +51,14 @@
...
 }
+
+.myft-section__column-content {
+	@include oGridColspan((default: full-width, M: 8, XL: 6));
+
+	@include oGridRespondTo(M) {
+		@include oGridCenter;
+	};
+
+	float: none;
```
Comment (GlynnPhillips) [inline client/components/v2/alerts/section.scss]: Do we need to float? I would have thought o-grid handled this but maybe I am wrong

Comment (debugwand) [inline client/components/v2/alerts/section.scss]: are we resetting styles? can we avoid doing that?

Comment (leggsimon) [inline client/components/v2/alerts/section.scss]: Maybe there's a better way of doing it but with `float` this happens on full-width
screenshot 2017-03-15 13 56 49
Removing the `float` property seems to fix that.

**Pull request merged**